docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -92,3 +92,37 @@ A new Timeline opens with an automatically populated KQL query. The query contai
 `file.hash.sha256 : c207213257a63589b1e1bd2f459b47becd000c1af8ea7983dd9541aff145c3ba`.
 
 Using a KQL query that includes both the indicator and source event allows Timeline to find all events and alerts that have matching field-value pairs.
+
+[discrete]
+[[attach-indicator-to-case]]
+== Attach indicators to cases
+
+Provide more context to your investigations by attaching indicators to cases:
+
+. From the Indicators table and click the *More actions menu* *(…​)*. Alternatively, open an indicator's details, then select *Take action*.
+. Select one of the following:
+.. *Add to existing case*: From the *Select case* dialog box, select the case to which you want to attach the indicator. 
+.. *Add to new case*: Configure the case details. Refer to <<cases-ui-open>> to learn more about opening a new case. 
+
+The indicator is added to the case as a new comment. 
```

Comment:
A couple small Asciidoc fixes: the options should be bullets, not ordered steps, and the final line should be a continuation of step 2, not main body text. The bullets also have to be encased in an [open block](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#attach-to-ancestor-list) (`--`) to get the correct alignment. (I tested this syntax locally so it should work, but might want to double-check on your end too.)
```suggestion
. Select one of the following:
+
--
* *Add to existing case*: From the *Select case* dialog box, select the case to which you want to attach the indicator. 
* *Add to new case*: Configure the case details. Refer to <<cases-ui-open>> to learn more about opening a new case. 
--
+
The indicator is added to the case as a new comment. 
```